<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>362</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>开始</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
color:black;</string>
  </property>
  <widget class="QWidget" name="log_in_widget" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>371</width>
     <height>281</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(151, 199, 255)</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font-size:30px;</string>
    </property>
    <property name="text">
     <string>超级智能选课机</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_account">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>148</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: grey;
background-color: white;
font-size: 14px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>请输入用户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_password">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>148</width>
      <height>23</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;your password&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: grey;
background-color: white;
font-size: 14px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>请输入密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_complete">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font-size: 18px;</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
